app/assets/stylesheets/sass/users_styles/show.scss

```diff
@@ -1,7 +1,7 @@
-.user__container--outer {
+.main-content__container--outer {
```

Comment:
Selector `main-content__container--outer` should be written in lowercase with hyphens